### DISCUSSION TITLE
Find Missing phashes for scenes

### DISCUSSION THREAD

**Discussion by Ashkaan** [stashapp/stash#3660] [Q&A] [answered]
When searching for duplicates, I get a message that says:

**"Missing phashes for 5 scenes. Please run the phash generation task."**

I've run the generation task, but there are 5 files that do not seem to generate.  How can I figure out which files those are?

**Comment by Flashy78** [comment, marked as answer]
Filter for `PHash is null`.

**Comment by Ashkaan** [comment]
Sadly, it comes up empty:

<img width="856" alt="image" src="https://user-images.githubusercontent.com/5667978/231559109-09c8692c-e5d8-4c7a-a27d-7f2cab2c8395.png">

**Comment by Flashy78** [comment]
Run a generate and look in the logs for any errors.
You can also join Discord to get more in depth help.

**Comment by Ashkaan** [comment]
Ok, thank you!

**Comment by IAmKontrast** [comment]
I had a similar problem but I get some scenes with PHash is null filter. All of the results had more than one file attached. Removing the duplicate files also solved my "Missing phashes for scenes" counter problem.

**Comment by Ashkaan** [comment]
Mine corrected itself after a schema update.
